polyfill/index.d.ts

```diff
@@ -471,8 +471,8 @@ export namespace Temporal {
     with(durationLike: DurationLike): Temporal.Duration;
     add(other: Temporal.Duration | DurationLike | string, options?: DurationRoundOptions): Temporal.Duration;
     subtract(other: Temporal.Duration | DurationLike | string, options?: DurationRoundOptions): Temporal.Duration;
-    round(options: DurationRoundOptions): Temporal.Duration;
-    total(options: DurationTotalOptions): number;
+    round(roundTo: string | DurationRoundOptions): Temporal.Duration;
```

Comment:
For example, we could rename these to `DurationRoundConfiguration` and `DurationTotalConfiguration`.

Reply:
Oops, I forgot about renaming the types. I'd suggest `DurationRoundTo` and `DurationTotalOf` here, and `RoundTo` for other types. A common convention I've seen in TS is when a type is used in only one context, then you try to name is as close as possible (albeit PascalCased) to the thing you're adding types to. 

Unless anyone objects, I'll make these name-changes in my next push.

Reply:
BTW, the options typings for `Duration.p.add` and `Duration.p.subtract` are wrong. They claim to be using round options but they don't; they only use `relativeTo`. 

I'll fix that too while I'm in the neighborhood.

Reply:
types for the `relativeTo` prop used in `Duration.p.total` was also wrong. Fixing too.